apps/definition_dictionary/lib/dictionary/type/map.ex

```diff
@@ -47,6 +47,8 @@ defmodule Dictionary.Type.Map do
   end
 
   defimpl Dictionary.Type.Normalizer, for: __MODULE__ do
+    def normalize(_, nil), do: Ok.ok(nil)
```

Comment:
Since we are passing null through for EVERY type, is there a way we could arrest this earlier and skip transforming nil values?

Reply:
Right now null is being converted to an empty string for Date and TimeStamp (I can't tell you if there's a good reason for that or not). Otherwise you could try to catch this in the `Dictionary` module. It might be nice to go ahead and keep it piecemeal anyway until the system has matured a little more.